clients/python/README.md

```diff
@@ -127,9 +126,9 @@ DEFAULT_CONFIG = {
     "start_publishing_thread": True,
 
     # how often should the events being sent to samsara
-    # in milliseconds
+    # in seconds
```

Comment:
if there is no good reason for which we have to move this to seconds i rather keep all the timers in millis for uniformity across the clients.